suzieq/sqobjects/basicobj.py

```diff
@@ -320,7 +321,7 @@ def get_table_info(self, table: str, **kwargs) -> pd.DataFrame:
 
         return self.engine.get_table_info(table, **kwargs)
 
-    def humanize_fields(self, df: pd.DataFrame, subset=None) -> pd.DataFrame:
+    def humanize_fields(self, df: pd.DataFrame, _=None) -> pd.DataFrame:
```

Comment:
No one is using this attribute, what its role is? Why don't we remove it?

Reply:
Lets do it the next time. I agree that we don't need it.